src/libraries/System.Private.CoreLib/src/System/Reflection/Emit/ConstructorOnTypeBuilderInstantiation.cs

```diff
@@ -57,7 +54,6 @@ public override int MetadataToken
                 }
                 else
                 {
```

Comment:
This whole method can be simplified to just `public override int MetadataToken => _ctor.MetadataToken;`

Reply:
(Dtto for other files.)